ext/RMagick/extconf.rb

```diff
@@ -130,6 +130,11 @@ def configure_compile_options
 
     end
 
+    def cant_install_rmagick?(version)
```

Comment:
It would be more natural to define a positive method (`can_install?`) and then negate its usage if necessary.